### ISSUE TITLE
How to enable zoom in top and bottom panels of timeline?

### ISSUE THREAD

**Issue by IsaacHub** [almende/vis#2920]
[http://visjs.org/examples/timeline/other/customTimeBars.html](http://visjs.org/examples/timeline/other/customTimeBars.html)
As you can see in this example the zoom functionality is enabled and working on panel `.vis-panel.vis-center`. In the same way how to zoom while hovering from `.vis-panel.vis-top` and `.vis-panel.vis-bottom` ?

We need this because UX is important, so that visitor won't be struggle with getting zoom, sometimes they may try to scroll while hovering the pointer above the top or bottom panels without aware, they may become frustrate and may think some bug has occurred or something, _(It happened to me)_

**Comment by jgorene** [comment]
Hello,
It seems to me that the example displayed is perhaps not the most appropriate for what you would like to get a better UX for your visitors.
I would rather look at this [example](http://visjs.org/examples/timeline/interaction/animateWindow.html) which suggests some interesting ways to improve the UX which perhaps will better suit your request...
Hoping;)

**Comment by IsaacHub** [comment]
@jgorene Nope.. That's NOT what I want.. I opened this issue as a feature request, by making both top and bottom panels scrollable without hiding them..

**Comment by JackD83** [comment]
I used this to scroll on the bottom panel:
$(".vis-panel.vis-bottom").bind('wheel', function (event) {    
                        var newEvent = new WheelEvent(event.originalEvent.type, event.originalEvent);
                        $(".vis-panel.vis-center").get(0).dispatchEvent(newEvent);                         
                    });

Pinch zoom is working fine

**Comment by cipri-tom** [comment]
thank you, @JackD83 !!

For those not using jQuery, it can be adapted to this:

```javascript
          var timelineCenterPanel = document.querySelector(".vis-panel.vis-center");
          document.querySelector('.vis-panel.vis-bottom').addEventListener('wheel', function scrollVisBottom(ev) {
            ev.stopPropagation();
            ev.preventDefault();
            var newEvent = new WheelEvent(ev.type, ev);
            timelineCenterPanel.dispatchEvent(newEvent);
          });
```

**Comment by mikhailswift** [comment]
It was very important for a usage I have of the timeline for .vis-bottom to be scrollable.  I forked vis and added the event handler to the bottom div, if this is something that is wanted I can create a PR to develop.  Reference: https://github.com/almende/vis/compare/master...mikhailswift:master

**Comment by neon-ninja** [comment]
Neither @JackD83 nor @cipri-tom 's snippets worked for me, so I had to do this:
```
$(".vis-panel.vis-bottom").bind('wheel', function (event) {
    console.log("scroll on bottom");
    if (event.originalEvent.deltaY < 0) {
        timeline.zoomIn(1);
    } else {
        timeline.zoomOut(1);
    }
});
```
